client/src/pages/Home.js

```diff
@@ -15,7 +16,51 @@ const Home = () => {
 	const [outputValue, setOutputValue] = useState("");
 	const [isMobile, setIsMobile] = useState(false);
 	const [alternativeValue, setAlternativeValue] = useState("");
-	const [showAlternatives, setShowAlternatives] = useState(false);
+  const [showAlternatives, setShowAlternatives] = useState(false);
+
+	const { user } = useAuth0();
+
+	const saveUserData = (userData) => {
+		// Send the user data to the backend
+		fetch("/api/saveUserData", {
+			method: "POST",
+			headers: {
+				"Content-Type": "application/json",
+			},
+			body: JSON.stringify(userData),
+		})
+			.then((response) => {
+				if (response.ok) {
+					return response.json(); // Parse the response data as JSON
+				} else {
+					throw new Error("Failed to save user data");
+				}
+			})
+			.then((data) => {
+				console.log(data.message); // Log the response message from the backend
+			})
+			.catch((error) => {
+				console.error("Error saving user data:", error);
+			});
+	};
+
+	useEffect(() => {
+		// Check if the user is authenticated
+		if (user) {
+			// Extract the necessary user data
+			const userData = {
+				name: user.name,
+				email: user.email,
+				sub: user.sub,
+			};
+			console.log(userData);
+
+			saveUserData(userData);
+		}
+	}, [user]);
```

Comment:
I don't think this is necessary, this will try and save to the database every time a user logs in or out, but we don't really need to do anything until the user presses save

Reply:
True, it was for before we made the decision to add a save button.